core/keyserver/keyserver.go

```diff
@@ -41,6 +41,7 @@ import (
 const (
 	directoryIDLabel = "directoryid"
 	logIDLabel       = "logid"
+	stageIDLabel     = "stage"
```

Comment:
Why is this label needed? Do you expect `watermarkWritten` to have at least one `stage` other than "written"? Same for other metrics - so far each of them has one stage with the same name as the metric.

Reply:
The long and the short is that I'm having trouble building a graph based in these three metrics: watermark_written, watermark_defined, watermark_applied. 

I had originally thought I could just graph them as three lines without difficulty. 
In practice, these three metrics get combined into one time series (one line), so I need to mark them separately so there's something to break them apart. That's one strategy - perhaps there is another way. @AlCutter ideas?

Reply:
We figured out a way to not need this